molecule/default/tasks/accprovision_test.yml

```diff
@@ -0,0 +1,19 @@
+---
+- name: Create the aci.accprov/v1alpha1.AccProvision
+  k8s:
+    state: present
+    namespace: '{{ namespace }}'
+    definition: "{{ lookup('template', '/'.join([samples_dir, cr_file])) | from_yaml }}"
+    wait: yes
+    wait_timeout: 300
+    wait_condition:
+      type: Running
+      reason: Successful
+      status: "True"
+  vars:
+    cr_file: 'aci_v1alpha1_accprovision.yaml'
+
+- name: Add assertions here
```

Comment:
is this task required?

Reply:
The molecule directory is a placeholder for future tests to be added